source/border_router.cpp

```diff
@@ -35,7 +35,8 @@
 #define STR_HELPER(x) #x
 #define STR(x) STR_HELPER(x)
 
-#define APP_DEFINED_HEAP_SIZE 32500
+#define APP_DEFINED_HEAP_SIZE 128500
```

Comment:
This rolls over to 62964 because the nsdynmem size values are 16bits.

Please use 65535. (or 0xffff)